src/main/scala/nl.knaw.dans.easy.validatebag/rules/metadata/package.scala

```diff
@@ -512,8 +518,8 @@ package object metadata extends DebugEnhancedLogging {
           else set.mkString("{", ", ", "}")
         }
 
-        lazy val onlyInBag = stringDiff(payloadPaths, pathsInFileXml)
-        lazy val onlyInFilesXml = stringDiff(pathsInFileXml, payloadPaths)
+        lazy val onlyInBag = stringDiff(payloadAndPreStagedFilePaths, pathsInFileXml)
```

Comment:
You can't really split the message to `only in payload` and `only in pre-staged.csv` because together they have to match the filepaths in filesXml.

I left the reading of payload files as it is now (walking the file system), because the code is much more simple that way.  If we would read the manifest file, the code would be something like this:
`t.tryBag.map(bag => bag.getPayLoadManifests.asScala.find(_.getAlgorithm == SHA1).map(manifest =>   manifest.getFileToChecksumMap.keySet().asScala.map(p => t.bagDir.relativize(p)).toSet))`